src/Core/src/Platform/iOS/SliderExtensions.cs

```diff
@@ -47,10 +48,13 @@ public static async Task UpdateThumbImageSourceAsync(this UISlider uiSlider, ISl
 		{
 			var thumbImageSource = slider.ThumbImageSource;
 
+			uiSlider.GetDisplayDensity();
```

Comment:
Does this do anything?

Reply:
I copied this line from Matt's PR. It calculates the density like so: https://github.com/dotnet/maui/blob/8289030e528cbc7737a35e61a91d978d6cb4b829/src/Core/src/Platform/iOS/ViewExtensions.cs#L911-L912

In theory, I think it does something. I can remove the change if you want.